README.md

```diff
@@ -144,21 +144,72 @@ Afterwards, when you run your environment e.g. with `docker-compose up stablex`
-- *SOLVER_TIME_LIMIT*: The offset from the start of the batch to cap the solver's execution time in seconds.
-- *SCHEDULER*: The scheduling method to use for solving batches. Can be either `system` (default) to use system time to schedule solving batches, or `evm` to use data read from the EVM in order to schedule solving batches.
-- *PRICE_SOURCE_UPDATE_INTERVAL*: Time interval in seconds in which price sources should be updated.
+The binary can be configured via command line options and environment variables: `cargo run -- --help`
+
+```
+Gnosis Exchange protocol driver.
+
+USAGE:
+    driver [OPTIONS] --network-id <network-id> --node-url <node-url> --private-key <private-key>
+
+FLAGS:
+    -h, --help
+            Prints help information
+
+    -V, --version
+            Prints version information
+
+
+OPTIONS:
+        --auction-data-page-size <auction-data-page-size>
+            The page size with which to read orders from the smart contract [env: AUCTION_DATA_PAGE_SIZE=]  [default:
+            100]
+        --http-timeout <http-timeout>
+            The default timeout in milliseconds of HTTP requests to remote services such as the Gnosis Safe gas station
+            and exchange REST APIs for fetching price estimates [env: HTTP_TIMEOUT=]  [default: 10000]
+        --log-filter <log-filter>
+            The log fiter to use.
+
+            This follows the `slog-envlogger` syntax (e.g. 'info,driver=debug'). [env: DFUSION_LOG=]  [default: info]
+    -i, --network-id <network-id>
+            The network ID used for signing transactions (e.g. 1 for mainnet, 4 for rinkeby, 5777 for ganache) [env:
+            NETWORK_ID=]
+    -n, --node-url <node-url>
+            The Ethereum node URL to connect to. Make sure that the node allows for queries witout a gas limit to be
+            able to fetch the orderbook [env: ETHEREUM_NODE_URL=]
+        --orderbook-filter <orderbook-filter>
+            JSON encoded object of which tokens/orders to ignore.
+
+            For example: '{ "tokens": [1, 2], "users": { "0x7b60655Ca240AC6c76dD29c13C45BEd969Ee6F0A": { "OrderIds": [0,
+            1] }, "0x7b60655Ca240AC6c76dD29c13C45BEd969Ee6F0B": "All" } }' [env: ORDERBOOK_FILTER=]  [default: {}]
+        --price-source-update-interval <price-source-update-interval>
+            Time interval in seconds in which price sources should be updated [env: PRICE_SOURCE_UPDATE_INTERVAL=]
+            [default: 300]
+    -k, --private-key <private-key>
+            The private key used by the driver to sign transactions [env: PRIVATE_KEY]
+
+        --rpc-timeout <rpc-timeout>
+            The timeout in milliseconds of web3 JSON RPC calls, defaults to 10000ms [env: WEB3_RPC_TIMEOUT=]  [default:
+            10000]
+        --scheduler <scheduler>
+            The kind of scheduler to use [env: SCHEDULER=]  [default: system]
+
```

Comment:
Are these seemingly random extra new-lines on purpose?

Reply:
That's the output of --help. I don't know why it outputs them like this but I'd rather have them in than remember to fix this manually.